### ISSUE TITLE
Toolbar buttons not usable via keyboard

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#4899]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.223
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
Get keyboard focus on a toolbar.

On Server, you can get focus to the toolbar via keyboard by using Ctrl+Alt+F to select the menubar, then use Tab to progress to the toolbar.

On Desktop, you cannot get focus to the toolbar in a consistent fashion (this will be tracked separately); to test, click on a blank area of the toolbar to give it focus (not visually indicated), then use Tab or Shift+Tab to move to buttons.

Focus is not visually indicated on all browsers currently; this issue will be tracked separately. For now, repro this on Chrome or RStudio Desktop.

On Desktop, 

### Describe the problem in detail
With focus on a button or menubutton (i.e. a button that causes a menu to appear, such as New File (first button on the main toolbar), hit Enter key or Spacebar. Nothing happens.

### Describe the behavior you expected
Enter key and spacebar must activate the button, per basic keyboard accessibility guidelines.

https://www.w3.org/TR/wai-aria-practices/#button

**Comment by rich-rstudio** [comment]
verified on desktop Version 1.3.921
server Version 1.3.926-1
